contracts/src/CrossChainApplications/examples/NativeTokenBridge/NativeTokenDestination.sol

```diff
@@ -125,21 +141,30 @@ contract NativeTokenDestination is TeleporterOwnerUpgradeable, INativeTokenDesti
         // Burn native token by sending to BURN_FOR_TRANSFER_ADDRESS
         Address.sendValue(payable(BURN_FOR_TRANSFER_ADDRESS), value);
 
+        uint256 scaledAmount;
+        if (multiplyOnSend) {
+            scaledAmount = msg.value * tokenMultiplier;
```

Comment:
can use `value` instead of `msg.value`. Can we move this before the `sendValue` call? I like to keep internal calls before external calls when possible